scheduler/src/cook/mesos/rebalancer.clj

```diff
@@ -375,10 +375,35 @@
                               (apply max))]
     utilization))
 
+(defn read-datomic-params
+  [conn]
+  (-<>>
+   (d/pull (mt/db conn) ["*"] :rebalancer/config)
+   (dissoc <> ":db/id" ":db/ident")
+   (map (fn [[k v]]
+          [(keyword (name (keyword k))) v]))
```

Comment:
Yes.  FYI this is actually existing code.  It's just been extracted from a previously inline function.